go.sum

```diff
@@ -1,18 +1,89 @@
+github.com/alcortesm/tgz v0.0.0-20161220082320-9c5fe88206d7 h1:uSoVVbwJiQipAclBbw+8quDsfcvFjOpI5iCf4p/cqCs=
```

Comment:
I'm impressed about the amount of dependencies of go-git, it seems ok for you ?

Reply:
Apparently the authors of the go-git did not implement some functionality and used the libraries created by the community. I don't see anything wrong with that.